packages/oscal-react-library/src/components/OSCALCatalogGroup.js

```diff
@@ -133,59 +216,85 @@ function OSCALCatalogControlListItem(props) {
         urlFragment={urlFragment}
         fragmentPrefix={appendToFragmentPrefix(fragmentPrefix, control.id)}
         fragmentSuffix={shiftFragmentSuffix(fragmentSuffix)}
+        previousHandledFragment={previousHandledFragment}
+        setPreviousHandledFragment={setPreviousHandledFragment}
       />
     </CollapsibleListItem>
-  ) : (
-    <StyledListItemPaper>
-      <StyledListItem>
-        <WithdrawnListItemText primary={itemText} withdrawn={withdrawn} />
-      </StyledListItem>
-    </StyledListItemPaper>
   );
 }
 
 function OSCALCatalogGroupList(props) {
   const {
     group,
-    control,
     urlFragment,
     fragmentPrefix,
     fragmentSuffix,
     isControlListItemOpened,
     setIsControlListItemOpened,
+    previousHandledFragment,
+    setPreviousHandledFragment,
   } = props;
-
+  const [isListItemNavigatedTo, isSetListItemNavigatedTo] = React.useState(false);
```

Comment:
```suggestion
  const [isListItemNavigatedTo, setIsListItemNavigatedTo] = React.useState(false);
```

Is this not more standard?